theseus/utils/examples/motion_planning/motion_planner.py

```diff
@@ -171,6 +172,9 @@ def __init__(
                     )
                 )
             )
+        # Add any extra user-provided cost functions before creating optimizer
+        if extra_costs_callback is not None:
+            extra_costs_callback(objective)
```

Comment:
We could add a warning if no extra costs were added (i.e., num of cost functions is the same before and after)?

Reply:
Should we also be disabling the goal cost if we do this?